atlas-starter.py

```diff
@@ -0,0 +1,91 @@
+import pymongo
+
+# Replace the placeholder data with your Atlas connection string. Be sure it includes
+# a valid username and password! Note that in a production environment,
+# you should not store your password in plain-text here.
+
+client = pymongo.MongoClient(
+      "mongodb+srv://<username>:<password>@<cluster-name>/test?retryWrites=true&w=majority")
```

Comment:
nit: for consistency, use the placeholder here: https://github.com/mongodb-university/atlas_starter_java/blob/master/src/main/java/mongodb/Main.java#L54

Reply:
👍